backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/CreateExperimentRunHandler.java

```diff
@@ -209,108 +191,115 @@ private ExperimentRun getExperimentRunFromRequest(
+                      runValueMap.put("experiment_id", newExperimentRun.getExperimentId());
+                      runValueMap.put("name", newExperimentRun.getName());
+                      runValueMap.put("description", newExperimentRun.getDescription());
+                      runValueMap.put("date_created", newExperimentRun.getDateCreated());
+                      runValueMap.put("date_updated", newExperimentRun.getDateUpdated());
+                      runValueMap.put("start_time", newExperimentRun.getStartTime());
+                      runValueMap.put("end_time", newExperimentRun.getEndTime());
+                      runValueMap.put("code_version", newExperimentRun.getCodeVersion());
+                      runValueMap.put("job_id", newExperimentRun.getJobId());
+                      runValueMap.put("parent_id", newExperimentRun.getParentId());
+                      runValueMap.put("owner", newExperimentRun.getOwner());
+
+                      runValueMap.put("environment", null);
+                      runValueMap.put("deleted", false);
+                      runValueMap.put("created", false);
+
+                      // Created comma separated field names from keys of above map
+                      String[] fieldsArr = runValueMap.keySet().toArray(new String[0]);
+                      String commaFields = String.join(",", fieldsArr);
+
+                      StringBuilder queryStrBuilder =
+                          new StringBuilder("insert into experiment_run ( ")
+                              .append(commaFields)
+                              .append(") values (");
+
+                      // Created comma separated query bind arguments for the values
+                      // based on the
+                      // keys of
+                      // above the map
+                      // Ex: VALUES (:project_id, :experiment_id, :name) etc.
+                      String bindArguments =
+                          String.join(
+                              ",",
+                              Arrays.stream(fieldsArr).map(s -> ":" + s).toArray(String[]::new));
+
+                      queryStrBuilder.append(bindArguments);
+                      queryStrBuilder.append(" ) ");
+
+                      LOGGER.trace(
+                          "insert experiment run query string: " + queryStrBuilder.toString());
+                      var query = handleForTransaction.createUpdate(queryStrBuilder.toString());
+
+                      // Inserting fields arguments based on the keys and value of map
+                      for (Map.Entry<String, Object> objectEntry : runValueMap.entrySet()) {
+                        query.bind(objectEntry.getKey(), objectEntry.getValue());
+                      }
+                      int count = query.execute();
+                      LOGGER.trace("ExperimentRun Inserted : " + (count > 0));
+                    }))
```

Comment:
Race condition bug for this i have created separate PR https://github.com/VertaAI/modeldb/pull/2247 but for testing other test cases this is here